src/components/20-molecules/datepicker/index.js

```diff
@@ -481,6 +488,16 @@ class AXADatepicker extends NoShadowDOM {
 
     this.startDate = overrideDate(year, month, day, startDate);
 
+    if (
+      allowedyears &&
+      !allowedyears.includes(startDate.getFullYear()) &&
+      !allowedyears.includes(year)
+    ) {
+      // to avoid an empty datepicker at startup set a valid year
+      const [newStartYear] = allowedyears;
+      this.year = newStartYear;
+    }
+
```

Comment:
@markus-walther I think this part of code is not at the correct place. Why? Because if i change the html attribute "year" after everything is renderd nothing is happend.

Reply:
OK, after getting clarification from @danyball : there is only one place currently where the property `year` is written, and that is in the `initDate` method.

So, I suggest to move this part of the code - i.e. the big if block - to that method. I also suggest trying to make the complicated if condition simpler, if possible.